crates/turborepo-lib/src/run/summary/mod.rs

```diff
@@ -589,7 +589,7 @@ impl<'a> RunSummary<'a> {
         if self.monorepo {
             Ok(serde_json::to_string_pretty(&self)?)
         } else {
-            let monorepo_rsm: SinglePackageRunSummary<'a, '_> = (&*self).into();
+            let monorepo_rsm = SinglePackageRunSummary::from(&*self);
```

Comment:
`&*self` might look weird, but that's just so we can get a `&self` from a `&mut self`

Reply:
maybe leave this comment in code if it's not a normal Rust thing?